integreat_cms/cms/views/push_notifications/push_notification_sender.py

```diff
@@ -87,17 +87,30 @@ def send_pn(self, pnt):
         :rtype: ~requests.Response
         """
         if settings.DEBUG:
-            region_slug = Region.objects.get(
-                id=settings.TEST_BLOG_ID
```

Comment:
Well, we could set a "TEST_BLOG_SLUG" in the settings?

Reply:
sounds good.

Reply:
Changed.